### ISSUE TITLE
Allow 1% overlap steps.

### ISSUE THREAD

**Issue by Weissrolf** [caewok/fvtt-walled-templates#11]
Hi.

I am trying to make walled-templates auto-target in Pathfinder 5/10/5 (movement) style by using the "Overlap method". Unfortunately 80% works for circle and one orientation of cone, but not for the other type of cone. And 70% works both both types of cones, but not for circles. So maybe I could find some in between coverage (71-79%) that might work for both, but currently only steps of 10 are allowed.

**Comment by Weissrolf** [comment]
Forget what I wrote, the values I listed where for DF Template Enhancement. Walled templates might also need in between steps, but before we can try that there seems to be a bug with the overlap method that makes it fail to highlight grids even at 0.0 setting.

![](https://i.imgur.com/avUPJ9d.png)

**Comment by Weissrolf** [comment]
I tried the same in an empty PF2 world and there it works properly. So I will have to find out what breaks functionality in my existing PF2 world.

But the gist of it is that overlap values between 0.7 and 0.8 work, so it would be nice if you could change the current settings limitation from 0.1 to 0.01.

**Comment by Weissrolf** [comment]
I seem to have fixed my broken world where walled templates deformed (and there also is a conflict with a the PF2 TokenMagic Integration module that I disable for the time being).

Setting an overlap value of 0.75 seems to give me the templates I need now, so coming back to my original feature request: please allow value changes of 0.01 in the settings. :)

**Comment by caewok** [comment]
Okay, if I did it right, both v9 (version 0.3.5) and v10 (version 0.4.1) versions should now allow step increment in 0.01. 

PF2 TokenMagic Integration definitely interferes with walled templates; see my comment on your other issue. 

Circles are hard because of some weirdness about exactly where the circle perimeter lies. So it would not surprise me if they are still occasionally screwed up. Feel free to submit a new issue if you see more weirdness.

**Comment by Weissrolf** [comment]
Great, thanks! :)
